src/main/java/com/bp3x/raidbot/commands/LFGCommand.java

```diff
@@ -1,42 +1,67 @@
-        this.ownerCommand = false;
-        this.category = new Category("General");
-    }
-
-@Override
-    protected void execute(CommandEvent event) {
-        log.info("LFG command by: " + event.getAuthor().getName() + "#" + event.getAuthor().getDiscriminator());
-        log.info("Wants to schedule " + event.getArgs());
-
-        LFGEmbedBuilder builder = null;
-        try {
-            builder = new LFGEmbedBuilder(event);
-            Message success = event.getChannel().sendMessage(builder.build()).complete();
-
-        } catch (NullPointerException npe) {
-            log.error("LFGEmbedBuilder was null, did not initialize", npe);
-        } catch (RaidBotRuntimeException e) {
-            log.error("Caught RaidBotRuntimeException while creating LFGEmbedBuilder.", e);
-        }
-    }
-}
+package com.bp3x.raidbot.commands;
+
+import com.bp3x.raidbot.commands.util.Event;
+import com.bp3x.raidbot.commands.util.LFGEmbedBuilder;
+import com.bp3x.raidbot.util.RaidBotRuntimeException;
+import com.jagrosh.jdautilities.command.Command;
+import com.jagrosh.jdautilities.command.CommandEvent;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.requests.RestAction;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+
+/**
+ * LFG Command to schedule events
+ */
+public class LFGCommand extends Command {
+    private final Logger log = LoggerFactory.getLogger(LFGCommand.class);
+
+    private static HashMap<Event, Message> plannedEventsList = new HashMap<>();
+
+    public LFGCommand() {
+        this.name = "lfg";
+        this.help = "Use to schedule a event";
+        this.arguments = "<shortName>";
+        this.guildOnly = false;
+        this.ownerCommand = false;
+        this.category = new Category("General");
+    }
+
+    public HashMap<Event, Message> getPlannedEventsList() { return plannedEventsList; }
+
+    @Override
+    protected void execute(CommandEvent event) {
+        log.info("LFG command by: " + event.getAuthor().getName() + "#" + event.getAuthor().getDiscriminator());
+        log.info("Wants to schedule " + event.getArgs());
+
+        String[] args = event.getArgs().split("\\s+");
+        if (args.length > 0) {
+            if (Event.eventExists(args[0])) {
+                try {
+                    Event plannedEvent = new Event(args[0]);
+                    plannedEvent.setPlayerStatus(event.getMember(), Event.EventPlayerStatus.ACCEPTED);
+
+                    LFGEmbedBuilder builder = new LFGEmbedBuilder(plannedEvent);
+                    Message success = event.getChannel().sendMessage(builder.build()).complete();
+
+                    RestAction<Void> reactWhiteCheckMark = success.addReaction("✅");
+                    RestAction<Void> reactQuestion = success.addReaction("❓");
+                    RestAction<Void> reactCross = success.addReaction("❌");
+
+                    RestAction.allOf(reactWhiteCheckMark, reactQuestion, reactCross).queue();

```

Comment:
Nitpicking here but could we make the order of reactions match the listing in the embed?

Right now reactions are Yes, Maybe, No and the Embed lists Yes, No, Maybe.

Reply:
I am fine with whatever order so long as they match, we can discuss the order if needed.